<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegWindowClass</class>
 <widget class="QWidget" name="RegWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>288</width>
    <height>238</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RegWindow</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout" stretch="20,50">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>用户名</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="nameEdit"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="20,50">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>密码</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="passwdEdit"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="20,60">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>重复密码</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="repeatEdit"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QPushButton" name="confirmBtn">
             <property name="text">
              <string>确定</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="cancelBtn">
             <property name="text">
              <string>取消</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
